build.rs

```diff
@@ -1,4 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: © Contributors to the iceoryx-rs project
+// SPDX-FileContributor: Mathias Kraus
```

Comment:
@dkroenke what do you think, with or without `SPDX-FileContributor`?

Reply:
This is fine according to SPDX standard: https://spdx.github.io/spdx-spec/file-tags/

We can go this way in every file.